.env-example

```diff
@@ -0,0 +1,12 @@
+TELEGRAM_TOKEN=
+TELEGRAM_ADMIN=
+CONTAINER_NAME=docker-controller-bot
+TZ=Europe/Madrid
+#TELEGRAM_GROUP=
+#TELEGRAM_THREAD=1
+#TELEGRAM_NOTIFICATION_CHANNEL=
+#CHECK_UPDATES=1
+#CHECK_UPDATE_EVERY_HOURS=4
+#BUTTON_COLUMNS=2
+#LANGUAGE=ES
+#EXTENDED_MESSAGES=0
```

Comment:
For the simplicity of the project and the way I build the image, only what is under /src should appear in the code.

```
docker-controller-bot/
├── Dockerfile_local
├── docker-compose.yaml
└── src
    ├── LICENSE
    ├── README.md
    ├── config.py
    ├── docker-controller-bot.py
    └── locale
        ├── en.json
        ├── es.json
        └── nl.json
```

So this file should not be committed.